### ISSUE TITLE
Update protein-protein interaction data

### ISSUE THREAD

**Issue by rkenyon** [PATRIC3/patric3_website#390]
Current PPIs are out of date.  Will require some backend work based on data sources and the way we are storing the data.  Chunhong is looking into this.  She has discovered that there are nearly double the current amount now in the source databases.

**Comment by hyoo** [comment]
I already imported Chunhong's new data and indexed on solr. However, PPI interface still need old api, which we have only one instance that serves production servers. I will setup another instance and plug it to new dataset, so that we can test over UI.

**Comment by mshukla1** [comment]
The PPI data was updated. However, during the testing session we found that the interactors were not correctly mapped to the corresponding PATRIC ids. This results in the PPI graph not being shown on the PATRIC website. 

We suspect that ids are not mapped because the PPI data generation scripts probably uses old data API or the database to get mappings between uniprot ids and PATRIC ids. Need to update the scripts to get the correct mappings. 

Decided to move it to September release. 

-Maulik
